io-package.json

```diff
@@ -154,7 +154,7 @@
     ],
     "dependencies": [
       {
-        "js-controller": ">=2.0.0"
+        "js-controller": ">=3.3.0"
```

Comment:
Why this change? because we can or why?

Reply:
I haven't testet it with js-controller 2. Is that version still supported?

Reply:
lets phrase it that way: YOu just added new logics "like the others" and did ot use any js-controller 3.3 features (and also the adapter is not), so assumption is that it still works. YOu are right there is no formal proof but controller is never really breaking - it just adds features that you can use.

But yes lets add it - but needs to be in changelog then